### DISCUSSION TITLE
Welcome to MarkTool Discussions!

### DISCUSSION THREAD

**Discussion by FXLP** [FXLP/MarkTool#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GZUchenyun** [comment]
This online website cannot be used！

**Comment by pjwzg** [comment]
这是自动回复邮件。您好，你所发的邮件已收到。                                                                         联系人：王志刚  联系电话：
